tests/commands/envelope_tests.py

```diff
@@ -0,0 +1,107 @@
+# encoding=utf-8
+# Copyright © 2017 Dylan Baker
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the alot.commands.envelope module."""
+
+from __future__ import absolute_import
+import os
+import contextlib
+import shutil
+import tempfile
+import unittest
+
+import mock
+
+from alot.commands import envelope
+
+
+@contextlib.contextmanager
+def temporary_directory(suffix='', prefix='', dir=None):
```

Comment:
yes, it can enable/disable various patterns.
I'm not sure if you as a collaborator can also edit those already:
https://www.codacy.com/app/patricktotzke/alot/patterns/list